gdk/wizard/WizardChecker.py

```diff
@@ -0,0 +1,78 @@
+from gdk.wizard.ConfigEnum import ConfigEnum
+import re
+import ast
+
+
+class WizardChecker:
+    def is_valid_input(self, input_value, field):
+        """
+        Prompts the user of all the optional ConfigEnum of the gdk config file and updates the
+        field_map if their answer is valid  as the user answers the question to each prompt
+
+        Parameters
+        ----------
+            input_value(string): The value entered by the user for the field
+            field(string): The field name of the field in the gdk config file
+
+        Returns
+        -------
+            boolean: True if the input value is valid for the field, False otherwise.
+        """
+
+        switch = {
+            ConfigEnum.AUTHOR: self.check_author,
+            ConfigEnum.VERSION: self.check_version,
+            ConfigEnum.CUSTOM_BUILD_COMMAND: self.check_custom_build_command,
+            ConfigEnum.BUILD_SYSTEM: self.check_build_system,
+        }
```

Comment:
Nit: Move this `switch` out into `__init__` of `WizardChecker` as we don't need to construct `switch` every time a caller uses `is_valid_input()`.

Reply:
Nit: Also, all of these checks are returning boolean values. So, you can rename `check_author` to `is_valid_author`.

Reply:
Yes i just moved it. Should i change all the checks to is_valid ?